bundles/org.eclipse.ui.workbench/eclipseui/org/eclipse/ui/internal/Workbench.java

```diff
@@ -2699,7 +2705,8 @@ private static String buildCommandLine(String workspace) {
 	 * as the workspace location.
 	 *
 	 * @param workspacePath the new workspace location
-	 * @return {@link IApplication#EXIT_OK} or {@link IApplication#EXIT_RELAUNCH}
+	 * @return {@link IApplication#EXIT_OK} or {@link IApplication#EXIT_RELAUNCH} or
+	 *         <code>null</code>
```

Comment:
While the returning constants is self explanatory, what does "return null" mean for the caller? NO exit? Don't care? Leave me alone with my problems? 
Just imagine you would see this javadoc and not see the implementation. What you would think about this API?